apps/src/code-studio/components/libraries/LibraryIdCopier.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {useRef} from 'react';
+import style from './library.module.scss';
+import i18n from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+
+export default function LibraryIdCopier({channelId}: {channelId: string}) {
+  const textContainer = useRef<HTMLInputElement>(null);
+  function handleCopyClick() {
+    // textContainer.current will be null prior to render. This is ok since nothing will be
+    // clickable at that point anyway.
+    textContainer.current?.select();
+    navigator.clipboard.writeText(channelId);
```

Comment:
Nit: looks like we also have a `copyToClipboard` util that includes a fallback for legacy browsers -- I see this isn't [available until 13.1 in Safari,](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard) and our IT requirements page currently says 13.x) Might not be worth the additional complexity though.